src/main/java/fragments/FooterFragment.java

```diff
@@ -0,0 +1,53 @@
+package fragments;
+
+import common.BasePage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class FooterFragment extends BasePage {
+
+    private static final String SIGN_UP_FIELD = id("newsletterInputField");
+    private static final String SUBMIT_BUTTON = id("newsletterSubmitCTA");
+    private static final String ERROR_MESSAGE = id("signupForm-response");
+    private static final String TITLE = "//*[@id='footerRow1ContentContainer']//h4";
+    private static final String SING_UP_RECOMMENDATION = "//*[@id='footerRow1ContentContainer']//div/p";
```

Comment:
```suggestion
    private static final String SIGN_UP_CONTAINER= id("footerRow1ContentContainer");
    private static final String TITLE = SIGN_UP_CONTAINER + "//h4";
    private static final String SING_UP_RECOMMENDATION = SIGN_UP_CONTAINER + "//div/p";
```
Maybe it looks better?